deploy/olm-catalog/tektoncd-pipeline-operator/0.0.1/tektoncd-pipeline-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -3,6 +3,12 @@ kind: ClusterServiceVersion
 metadata:
   annotations:
     capabilities: Basic Install
+    categories: "Developer Tools"
+    description: An operator to install TektonCD Pipeline
+    containerImage: quay.io/rhpipeline/nikthoma_tektoncd-pipeline-operator:v0.2.0-2
+    createdAt: 2019-03-15T19:44:21Z
```

Comment:
shouldn't this be updated as well?

Reply:
updated using image from quay.io/openshift-pipeline org